toponetx/datasets/graph.py

```diff
@@ -9,7 +10,9 @@
 from toponetx.algorithms.spectrum import hodge_laplacian_eigenvectors
 from toponetx.transform.graph_to_simplicial_complex import graph_2_clique_complex
 
-__all__ = ["karate_club"]
+__all__ = ["karate_club", "coauthorship"]
```

Comment:
I am not sure the addition of this dataset to graph is justified, maybe it needs a different file all together. Can you justify your choice for graph? other people might not find it intuitive anyway,

Reply:
Yes, can you suggest a name for the file?
I'll be away for a week, I can fix the code based on the comments when I'm back.